test/camunda-cloud/ModelerSpec.js

```diff
@@ -168,7 +168,7 @@ describe('<CamundaCloudModeler>', function() {
       expect(modeler.get('copyPasteBehavior')).to.exist;
       expect(modeler.get('cleanUpBusinessRuleTaskBehavior')).to.exist;
       expect(modeler.get('createZeebeCallActivityBehavior')).to.exist;
-      expect(modeler.get('formDefinitionBehavior')).to.exist;
+      expect(modeler.get('formsBehavior')).to.exist;
```

Comment:
Do we still want the tests to be tightly-coupled to the behaviors modules? I don't see value in this.

Reply:
I don't see the value, too. What we may want to assert is the amount of modules, or actual features (via an integration test).

Reply:
My proposal is to not add tests for new modules in the behaviors package, but keep the already written as a sign of "camunda-bpmn-js-behaviors are included". I'd rather not duplicate tests from the behaviors module because it adds work without substantial improvement. We will see integration in action in the `bpmn-js-element-templates` module.

Reply:
Sounds great :+1: